README.md

```diff
@@ -108,7 +108,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <table>
   <tr>
     <td align="center"><a href="https://github.com/jeromelaban"><img src="https://avatars0.githubusercontent.com/u/5839577?v=4" width="100px;" alt="Jérôme Laban"/><br /><sub><b>Jérôme Laban</b></sub></a><br /><a href="https://github.com/unoplatform/uno/commits?author=jeromelaban" title="Code">💻</a> <a href="#content-jeromelaban" title="Content">🖋</a> <a href="https://github.com/unoplatform/uno/commits?author=jeromelaban" title="Documentation">📖</a> <a href="#example-jeromelaban" title="Examples">💡</a> <a href="#maintenance-jeromelaban" title="Maintenance">🚧</a> <a href="#infra-jeromelaban" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="#ideas-jeromelaban" title="Ideas, Planning, & Feedback">🤔</a> <a href="#review-jeromelaban" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/unoplatform/uno/commits?author=jeromelaban" title="Tests">⚠️</a> <a href="#projectManagement-jeromelaban" title="Project Management">📆</a></td>
-    <td align="center"><a href="http://www.dissolutegames.com/"><img src="https://avatars0.githubusercontent.com/u/8270914?v=4" width="100px;" alt="David Oliver"/><br /><sub><b>David Oliver</b></sub></a><br /><a href="#blog-davidjohnoliver" title="Blogposts">📝</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Code">💻</a> <a href="#content-davidjohnoliver" title="Content">🖋</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Documentation">📖</a> <a href="#maintenance-davidjohnoliver" title="Maintenance">🚧</a> <a href="#tutorial-davidjohnoliver" title="Tutorials">✅</a> <a href="#review-davidjohnoliver" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Tests">⚠️</a> <a href="#example-davidjohnoliver" title="Examples">💡</a></td>
+    <td align="center"><a href="https://www.dissolutegames.com/"><img src="https://avatars0.githubusercontent.com/u/8270914?v=4" width="100px;" alt="David Oliver"/><br /><sub><b>David Oliver</b></sub></a><br /><a href="#blog-davidjohnoliver" title="Blogposts">📝</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Code">💻</a> <a href="#content-davidjohnoliver" title="Content">🖋</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Documentation">📖</a> <a href="#maintenance-davidjohnoliver" title="Maintenance">🚧</a> <a href="#tutorial-davidjohnoliver" title="Tutorials">✅</a> <a href="#review-davidjohnoliver" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Tests">⚠️</a> <a href="#example-davidjohnoliver" title="Examples">💡</a></td>
```

Comment:
Do not modify this section, it is generated content, it will be overwritten.

Reply:
Agreed. The rest of the PR is good to go, so I'll merge this as-is and then next time we add a contributor to the documentation it'll undo this.